include/os/linux/zfs/sys/trace_defs.h

```diff
@@ -0,0 +1,30 @@
+#ifndef _OS_LINUX_ZFS_TRACE_H
+#define	_OS_LINUX_ZFS_TRACE_H
```

Comment:
Please add the short CDDL header to this new file, and some while space around the guard macro's so they match the other headers.,